### PR TITLE
Remove apparent reference to non-existent [[Get]] internal method

### LINKED ISSUE #2169: [[Get]] method doesn't exist in CredMan
[§5.1.4. Use an Existing Credential to Make an Assertion - PublicKeyCredential’s \[\[Get\]\](options) Method](https://w3c.github.io/webauthn/#sctn-getAssertion) appears to reference a `[[Get]]` internal method on the [`Credential` interface](https://w3c.github.io/webappsec-credential-management/#the-credential-interface) from CredMan, but no such internal method exists (unlike [`[[Create]]`, which does exist](https://w3c.github.io/webappsec-credential-management/#algorithm-create-cred)). Rather, [`[[DiscoverFromExternalSource]]`](https://w3c.github.io/webappsec-credential-management/#algorithm-discover-creds) is the actual internal method we override.

## Proposed Change

- Delete the heading [§5.1.4.1. PublicKeyCredential’s [[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors) Method](https://w3c.github.io/webauthn/#sctn-discover-from-external-source) (without changing any of the text around it).
- Rename the heading [§5.1.4. Use an Existing Credential to Make an Assertion - PublicKeyCredential’s \[\[Get\]\](options) Method](https://w3c.github.io/webauthn/#sctn-getAssertion) to **5.1.4. Use an Existing Credential to Make an Assertion - PublicKeyCredential’s [[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors) Internal Method**.
- For consistency, change "Method" to "Internal Method" in the heading [§5.1.3. Create a New Credential - PublicKeyCredential’s [[Create]](origin, options, sameOriginWithAncestors) Method](https://w3c.github.io/webauthn/#sctn-createCredential).
- Similarly, change "Method" to "Internal Method" in the heading [§5.1.5. Store an Existing Credential - PublicKeyCredential’s [[Store]](credential, sameOriginWithAncestors) Method](https://w3c.github.io/webauthn/#sctn-storeCredential).

### PR DESCRIPTION
**Pull request by emlun**
Closes #2169.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/2180.html" title="Last updated on Oct 7, 2024, 3:34 PM UTC (068d7f5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/2180/6744192...068d7f5.html" title="Last updated on Oct 7, 2024, 3:34 PM UTC (068d7f5)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -2301,7 +2311,10 @@ The following [=simple exceptions=] can be raised:
 
 </dl>
 
-### Use an Existing Credential to Make an Assertion - PublicKeyCredential's `[[Get]](options)` Method ### {#sctn-getAssertion}
+### Use an Existing Credential to Make an Assertion - PublicKeyCredential's <code><dfn for="PublicKeyCredential" method>[DISCOVER-METHOD-DEF]</dfn></code> Internal Method ### {#sctn-getAssertion}
```
Comment (nsatragno) [inline index.bs:2314]: The proposed text is more accurate, but now the discussion of why `[[CollectFromExternalSource]]` has a default implementation and `[[DiscoverFromExternalSource]]` does the heavy lifting (essentially, how webauthn hooks into `navigator.credentials.get`) is more spread out. Some of the discussion on the following paras is not directly related to `[[DiscoverFromExternalSource]]` (e.g. the conditional mediation bits).

I would prefer if this header remained, perhaps named

```suggestion
### Use an Existing Credential to Make an Assertion ### {#sctn-getAssertion}
```

And then bring back the heading on L2336 to cover the discussion of `[[DiscoverFromExternalSource]]` as a subsection.

Comment (emlun) [reply index.bs:2314]: Ok, I'll take another pass at this.

> Some of the discussion on the following paras is not directly related to `[[DiscoverFromExternalSource]]` (e.g. the conditional mediation bits).

Ah, good point!

**Review by nsatragno — APPROVED** [review]
Thank you!

**Comment by emlun** [comment]
Force-pushed to cut out commits 00fbea535b432f3b3ae6acc214000a603c7d9e95 - 639c93520c87636b7d9ab1b63b4c29d438002f2d whose combination was the empty diff.

**Review by timcappalli — APPROVED** [review]

**Review by akshayku — APPROVED** [review]

**Review by MasterKale — APPROVED** [review]

**Review by selfissued — APPROVED** [review]
Approved during the 9-Oct-24 call

**Pull request merged**